allennlp/models/ensemble/bidaf_ensemble.py

```diff
@@ -0,0 +1,109 @@
+from overrides import overrides
+import torch
+from typing import Dict, List, Any
+
+from allennlp.models.ensemble.ensemble import Ensemble
+from allennlp.models.archival import load_archive
+from allennlp.models import Model
+from allennlp.models.reading_comprehension import BidirectionalAttentionFlow
+from allennlp.common import Params
+from allennlp.data import Vocabulary
+from allennlp.training.metrics import SquadEmAndF1
+
+@Model.register("bidaf-ensemble")
+class BidafEnsemble(Ensemble):
+
+    def __init__(self,
+                 submodels: List[BidirectionalAttentionFlow]) -> None:
+        super(BidafEnsemble, self).__init__(submodels)
+
+        # Setting this propagates calls to .eval() so dropout is disabled on the submodels in evaluation
+        # and prediction.
+        self.linears = torch.nn.ModuleList(submodels)
+
+        self.submodels = submodels
+
+        self._squad_metrics = SquadEmAndF1()
+
+    @overrides
+    def forward(self,  # type: ignore
+                question: Dict[str, torch.LongTensor],
+                passage: Dict[str, torch.LongTensor],
+                span_start: torch.IntTensor = None,
+                span_end: torch.IntTensor = None,
+                metadata: List[Dict[str, Any]] = None) -> Dict[str, torch.Tensor]:
+
+        subresults = []
+        for i, submodel in enumerate(self.submodels):
+            subresults.append(submodel.forward(question, passage, span_start, span_end, metadata))
```

Comment:
I would just use a list comprehension here (and not use `enumerate`, since you're not using the index, it looks like)